policyengine_us/parameters/gov/states/vt/tax/income/credits/ctc/amount.yaml

```diff
@@ -0,0 +1,10 @@
+description: Vermont provides this maximum Child Tax Credit per qualifying child.
+values:
+  0000-01-01: 0
+  2023-01-01: 1_000
```

Comment:
the child tax credit is in the 2022 form: 
file:///Users/pavelmakarchuk/Desktop/PolicyEngine/Tax%20Forms/Vermont/Income%20Booklet-2022.pdf

also check 2021 form